addons/website/views/website_pages_views.xml

```diff
@@ -36,7 +36,7 @@
             <notebook>
                 <page string="Name">
                     <group class="mt-4">
-                        <field name="name" string="Page Name" widget="page_name" placeholder="e.g. Home Page"/>
```

Comment:
Note: just checked to be sure: should someone pull the code without updating the app, this does not crash, just a warning in console that says that widget does not exist :+1: